### PR TITLE
Docs-UG: Mention that for find command If no prefix is used for the first arg, it will be used to search name

### LINKED ISSUE #258: [PE-D][Tester C] `Find` command: Update to remove functionality of finding by `find keyword [KEYWORDS]`
In the UG and find command help message, the format given is `find [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [w/RANGE] [h/RANGE] [nt/NOTE] [t/TAG]…`. However, `find keyword [KEYWORDS]` is still allowed.

Not sure if this is expected behaviour. If it is, this report should be taken as a documentation bug report as the UG should be updated to specify that this behaviour is allowed.

Steps to reproduce (using default sample data):
1. `find al`

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: shaunlxw/ped#5

### PR DESCRIPTION
**Pull request by bgopi23**
Fixes #258

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -274,11 +274,19 @@ Format: `find [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [w/RANGE] [h/RANGE] [nt/N
...
 * All fields except `TAG` will be matched based on substring (e.g `Wen` will match `Wendy`)
+
 <div markdown="block" class="alert alert-warning">:warning: **Take note:**
 * Unlike other fields, `TAG` must be an exact match (case-insensitive)
 * E.g `find t/fri` will not match the tag `friend`
 * But `find t/fRieNd` will match the tag `friend`
 </div>
+<div markdown="span" class="alert alert-info">:bulb: **Tip:**
+* If no prefix is specified for the first argument, it will be used to search for a clients name
+* If prefix no prefix is specified to search for a client's name, it must be the first field to search
```
Comment (bgopi23) [inline docs/UserGuide.md:285]:
```suggestion
* If no prefix is specified to search for a client's name, it must be the first field to search
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -274,11 +274,19 @@ Format: `find [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [w/RANGE] [h/RANGE] [nt/N
...
+
 <div markdown="block" class="alert alert-warning">:warning: **Take note:**
 * Unlike other fields, `TAG` must be an exact match (case-insensitive)
 * E.g `find t/fri` will not match the tag `friend`
 * But `find t/fRieNd` will match the tag `friend`
 </div>
+<div markdown="span" class="alert alert-info">:bulb: **Tip:**
+* If no prefix is specified for the first argument, it will be used to search for a clients name
+* If no prefix is specified to search for a client's name, it must be the first field to search
+* E.g `find roy t/friends` is allowed
```
Comment (bgopi23) [inline docs/UserGuide.md:286]:
```suggestion
* e.g. `find roy t/friends` is allowed
```

**Review by ruijietay — APPROVED** [review]
LGTM

**Review by RyanNgWH — APPROVED** [review]
looking good!

**Pull request merged**